contrib/chatops/pack.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 name: chatops
 description: ChatOps integration pack
-version: 1.0.2
```

Comment:
Probably bump the version to 1.1.0 because it's a feature add.

Reply:
Sounds reasonable to me.  I'll bump the minor version.